src/frontends/tensorflow/src/helper_ops/undefined_constant.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "internal_operation.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+class UndefinedConstant : public InternalOperation {
```

Comment:
Why undefined constant is needed for assert support?

Reply:
The undefined constants serve to represent constants of types that are not supported by OV like string type. Without it the conversion fails to convert Constant nodes of such type that happens before we start to convert Assert node.

Reply:
Undefined const makes me think that it is a const undefined in tf, but what it actually is string, unsupported const. Can we rename it?

Reply:
Agree, let us rename it to `UnsupportedConstant`

Reply:
Done